src/main/java/com/devchang202/calculator/Calculator.java

```diff
@@ -0,0 +1,112 @@
+package com.devchang202.calculator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Calculator {
+    int add(int a, int b) {
+        return a + b;
+    }
+
+    int substrate (int a, int b){
```

Comment:
```suggestion
    int subtract(int a, int b) {
```

컨벤션을 위반하셨습니다. IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.